### PR TITLE
WebDriver conformance changes for Firefox 107

### PR DESCRIPTION
**Pull request by whimboo**
The list of all the changes in Firefox 107 can be found [here](https://bugzilla.mozilla.org/buglist.cgi?component=Agent&component=Marionette&component=WebDriver%20BiDi&j_top=OR&resolution=FIXED&o1=equals&query_format=advanced&f1=cf_status_firefox107&v1=fixed&product=Remote%20Protocol&product=Testing&list_id=16288684).

Can you please review these items @juliandescottes? Thanks

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/107/index.md [hunk 1]
```diff
@@ -58,7 +58,23 @@ This article provides information about the changes in Firefox 107 that will aff
 
 ### WebDriver conformance (WebDriver BiDi, Marionette)
 
-#### Removals
+#### WebDriver BiDi
+
+- Added Realm support to `target` argument for `script.evaluate`, `script.callFunction`, and `script.disown` commands ({{bug("1779231")}}).
```
Comment (juliandescottes) [inline files/en-us/mozilla/firefox/releases/107/index.md:63]: We can also mention that the commands are now officially enabled:
> [Bug 1778976 - Enable "script.evaluate", "script.callFunction" and "script.disown" on release channels](https://bugzilla.mozilla.org/show_bug.cgi?id=1778976)

**Review by juliandescottes — COMMENTED** [review]
Potentially one additional bug to mention, but other than this LGTM!

File: files/en-us/mozilla/firefox/releases/107/index.md (hunk 1 above)
Comment (whimboo) [reply files/en-us/mozilla/firefox/releases/107/index.md:63]: This was already done in Firefox 106 which was a late uplift.

**Comment by whimboo** [comment]
> Potentially one additional bug to mention, but other than this LGTM!

It was not necessary given that these referred to features already shipped in Firefox 106.

@Rumyra could you please review? Thanks

**Review by Rumyra — APPROVED** [review]

**Comment by Rumyra** [comment]
Great thanks @whimboo !

**Pull request merged**